meeting/meeting_january_11_2022.md

```diff
@@ -0,0 +1,57 @@
+# January 11th (7 PM CST)/10th (9 AM JST) Meeting 2022
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Chris)**
+* **#Query(Matt)**
+* **#Query(Tibo)**
+* **#Query(Jesse)**
+* **#Query(Ben)**
+* **#Query(Philipp)**
```

Comment:
Unfortunately I won't be able to make it as I'm in Europe and the call will be at 1am for me.